core/src/create.jl

```diff
@@ -85,7 +85,29 @@ end
 
 function Pump(db::DB, config::Config)::Pump
     static = load_structvector(db, config, PumpStaticV1)
-    return Pump(static.node_id, static.flow_rate)
+
+    control_mapping = Dict{Tuple{Int, String}, NamedTuple}()
+
+    if length(static.control_state) > 0 && !any(ismissing.(static.control_state))
+        # Starting flow_rates are first one found (can be updated by control initialisation)
```

Comment:
So the default state (normally missing) is not necessarily the one used, if the order of the static table is incorrect?

Reply:
I think the previous code making an `ismissing` mask (or a defaultmask, once we define an other default value), makes more sense, also in readability.